applications/luci-app-shadowsocks-libev/htdocs/luci-static/resources/shadowsocks-libev.js

```diff
@@ -238,5 +242,74 @@ return L.Class.extend({
 			window.open(L.url('admin/system/opkg') +
 				'?query=' + opkg_package, '_blank', 'noopener');
 		};
+	},
+	startsWith: function(str, search) {
```

Comment:
This could be dropped if you replace it with `str.indexOf(search) === 0` checks.